sdk/ml/azure-ai-ml/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 1.16.0 (unreleased)
 
 ### Features Added
+- Added support for Promptflow evaluators with `experimental` tag.
```

Comment:
Just a suggestion:

Add experimental support for working with Promptflow evaluators: `ml_client.evaluators`

Reply:
Fixed